src/EditorFeatures/CSharp/EmbeddedLanguages/CSharpJsonBraceMatcher.cs

```diff
@@ -12,13 +12,12 @@
 namespace Microsoft.CodeAnalysis.Editor.CSharp.EmbeddedLanguages
 {
     [ExportEmbeddedLanguageBraceMatcherInternal(
-        PredefinedEmbeddedLanguageBraceMatcherNames.Json, LanguageNames.CSharp, supportsUnannotatedAPIs: true, "Json"), Shared]
-    internal sealed class CSharpJsonEmbeddedLanguageBraceMatcher :
-        AbstractJsonEmbeddedLanguageBraceMatcher
+        PredefinedEmbeddedLanguageNames.Json, LanguageNames.CSharp, supportsUnannotatedAPIs: true, "Json"), Shared]
+    internal sealed class CSharpJsonBraceMatcher : AbstractJsonBraceMatcher
     {
         [ImportingConstructor]
         [Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
-        public CSharpJsonEmbeddedLanguageBraceMatcher()
+        public CSharpJsonBraceMatcher()
```

Comment:
moved to a simpler and more consistent naming pattern.  Instead of repeating 'EmbeddedLanguage' in teh name, we just state the name of the embedded language.  Regex/Json are already clearly embedded langauges, so this cut down on verbosity everywhere.